tests/test_layout.py

```diff
@@ -66,15 +63,17 @@ def test_search_change_column(self, base_url, selenium):
                 assert cs_super.are_search_results_found
                 assert cs_super.search_results_table_header.is_column_not_present(current_column)
 
+        assert 'product' == cs_super.field('0')
```

Comment:
This might benefit from a concise comment indicating what these asserts are for, given the functionality change you walked me through earlier today, but not blocking r+ on that at this time.